experiments/ClimaEarth/user_io/arg_parsing.jl

```diff
@@ -1,5 +1,21 @@
 import YAML
 
+abstract type AbstractModeType end
+abstract type AbstractSlabPlanetModeType <: AbstractModeType end
+abstract type AbstractSlabPlanetModeSubType1 <: AbstractSlabPlanetModeType end
```

Comment:
This needs a better name. It is the group of modes that is used for model setup [here](https://github.com/CliMA/ClimaCoupler.jl/blob/5318c3bc6723f0176036eef48758515b09ec9182/experiments/ClimaEarth/run_amip.jl#L337)

Reply:
Or it could just be removed

Reply:
In the model setup, we could dispatch off of `AbstractSlabPlanetModeType`, and have a special case just for `SlabPlanetEisenman`